lib/bootloader_setup.pm

```diff
@@ -403,6 +403,10 @@ sub specific_bootmenu_params {
         $args .= " addon=" . $addon_url;
     }
 
+    if (check_var('ISO_IN_EXTERNAL_DRIVE', 1)) {
+        $args .= "install=hd:/install.iso";
```

Comment:
It might be safer to start the concatenation of `$args` with ` ` (space), as are other concatenations.